packages/remix/test/integration/test/client/root-loader.test.ts

```diff
@@ -0,0 +1,183 @@
+import { test, expect, Page } from '@playwright/test';
+
+export function getRouteData(page: Page): Promise<any> {
+  return page.evaluate('window.__remixContext.routeData').catch(err => {
+    return {};
+  });
+}
+
+test('should inject `sentry-trace` and `baggage` into root loader returning `{}`.', async ({ page }) => {
+  await page.goto('/?type=empty');
+  const sentryTraceTag = await page.$('meta[name="sentry-trace"]');
+  const sentryTraceContent = await sentryTraceTag?.getAttribute('content');
+
+  expect(sentryTraceContent).toEqual(expect.any(String));
+
+  const sentryBaggageTag = await page.$('meta[name="baggage"]');
```

Comment:
Can we extract out the `meta[name=` getter logic into util functions to help clean up the function bodies?